components/family-studies/ui/src/main/resources/PhenoTips/FamilySelector.xml

```diff
@@ -125,7 +125,7 @@
   widgets.FamilySelector = Class.create({
 
         initialize: function() {
-            console.log("init");
+            console &amp;&amp; console.log("init");
```

Comment:
I'm sure all logs in FamilySelector can be just removed. Sorry I did not commit my changes (since I already did that), was just playing more with IE9